projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.spec.ts

```diff
@@ -143,7 +143,7 @@ describe('TabParagraphContainerComponent', () => {
     component.ngOnInit();
     // reset the state
     windowRef.nativeWindow.history.replaceState(null, null);
-    expect(component.activeTabNum).toEqual(1);
+    expect(component.activeTabNum).toEqual(1 || -1);
```

Comment:
What this addition does? I believe `1 || -1` is just `1`. This `expect` doesn't test if activeTabNum is 1 or -1, I believe.